packages/react-ui-ag/src/Gmap/Markers.js

```diff
@@ -2,29 +2,34 @@ import React, { PureComponent } from 'react'
 import PropTypes from 'prop-types'
 import get from 'lodash/get'
 import { Markers as Gmarkers } from '@rentpath/react-ui-core'
-import {
-  redDotIcon,
-  greyDotIcon,
-  blackDotIcon,
-} from './markerIcons'
+import MarkerIcons from './markerIcons'
 
 const NOOP = () => ({})
 
+const markerIcons = [
+  'greyDotIcon',
+  'redDotIcon',
+  'blackDotIcon',
+  'blackDotIconWithBalloon',
+  'redPdpMapBalloon',
+  'redPdpMapBalloonWithBlackDot',
+]
+
 export default class Markers extends PureComponent {
   static propTypes = {
     marker: PropTypes.func,
-    markerIcon: PropTypes.func,
-    markerInactiveIcon: PropTypes.func,
-    markerIconHover: PropTypes.func,
+    markerIcon: PropTypes.oneOf(markerIcons),
+    markerInactiveIcon: PropTypes.oneOf(markerIcons),
+    markerIconHover: PropTypes.oneOf(markerIcons),
```

Comment:
I wonder if `_.keys(MarkerIcons)` would work here? That way we wouldn't have to maintain a list of keys